tests/units/Transvision/Utils.php

```diff
@@ -458,12 +458,14 @@ public function redirectToAPIDP()
     {
         return [
             [
-                'http://transvision.mozfr.org/string/?entity=browser/chrome/browser/downloads/downloads.properties:stateStarting&repo=central',
-                'http://transvision.mozfr.org/string/?entity=browser/chrome/browser/downloads/downloads.properties:stateStarting&repo=central&json',
```

Comment:
With the removal of these tests that contained slashes in entity names, it actually hides the fact that your new version of the function returns escaped slashes instead of real slashes.